### PR TITLE
FF Q2 2024

### PR DESCRIPTION
**Pull request by aznrayizzle**

### CODE REVIEW

**Review by arsulegai — APPROVED** [review]

**Review by petermetz — APPROVED** [review]

**Review by tkuhrt — COMMENTED** [review]
All images should go into a 2024 folder instead of the 2023 folder.

File: project-reports/2024/2024-Q2-Hyperledger-FireFly.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: 2024 Q42 Hyperledger FireFly
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-FireFly.md:3]:
```suggestion
title: 2024 Q2 Hyperledger FireFly
```

File: project-reports/2024/2024-Q2-Hyperledger-FireFly.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+- Duke Web3 Conference: [Real World Asset Tokenization](https://www.dukeweb3conference.com/speakers)
+- Token 2049: [Decentralized Trust & Web3 in the Enterprise](https://lu.ma/lxjqfuw3?hss_channel=lcp-10851358)
+- University of Southampton Business School lecture: [Web3 in Industry](https://www.linkedin.com/feed/update/urn:li:activity:7196499141403856896)
+- [FireFly 1.3 webinar](https://www.meetup.com/hyperledger-raleigh/events/301138155/) scheduled for 6/10
+
+### Contributor activity updates
+
+The star activity for the FireFly repository has continued to grow from 430 end of last quarter to 481 end of this quarter. Discord activity remains consistent with ~45 DAU.
+
+![alt_text](images/HLFF_2024_Q2_stars.png "image_tooltip")
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-FireFly.md:39]: Looks like path does not match location of file.

File: project-reports/2024/2024-Q2-Hyperledger-FireFly.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+![alt_text](images/HLFF_2024_Q2_1.png "image_tooltip")
+
+![alt_text](images/HLFF_2024_Q2_2.png "image_tooltip")
+
+![alt_text](images/HLFF_2024_Q2_3.png "image_tooltip")
+
+![alt_text](images/HLFF_2024_Q2_4.png "image_tooltip")
+
+![alt_text](images/HLFF_2024_Q2_5.png "image_tooltip")
```
Comment (tkuhrt) [inline project-reports/2024/2024-Q2-Hyperledger-FireFly.md:95]: location of image differs from that specified here.

**Review by tkuhrt — CHANGES_REQUESTED** [review]

**Review by denyeart — APPROVED** [review]

**Review by jimthematrix — APPROVED** [review]

File: project-reports/2024/2024-Q2-Hyperledger-FireFly.md (hunk 2 above)
Comment (VRamakrishna) [reply project-reports/2024/2024-Q2-Hyperledger-FireFly.md:39]: The `images` folder was pushed to `2023` instead of `2024`.

**Review by swcurran — APPROVED** [review]

**Review by mbrandenburger — APPROVED** [review]

**Review by tkuhrt — APPROVED** [review]

**Pull request merged**